source/content/guides/drupal-9-migration/03-upgrade-to-d9.md

```diff
@@ -18,11 +18,7 @@ This doc shows how to upgrade an existing Pantheon-hosted Drupal 8 site without
 
 ## Overview
 
-The goals of this upgrade are to:
-
-- set the Drupal core dependency to Drupal 9, and
-
-- have Composer manage those dependencies in the new site, instead.
+The goals of this upgrade are to set the Drupal core dependency to Drupal 9, and have Composer manage those dependencies in the new site.
```

Comment:
This sentence is a little unclear. Consider rewording: The goal of this upgrade is to set the Drupal core dependency to Drupal 9. This allows Composer to manage core dependencies in the new site.